content/02.main-text.md

```diff
@@ -20,8 +20,54 @@ Although we requested that some authors participate for their specific expertise
+
+Manubot uses [Pandoc](https://pandoc.org/) to convert manuscripts from markdown source to HTML (and optionally DOCX) output.
+Pandoc supports conversion between a large number of formats, offering Manubot users broad interoperability and preventing platform lock-in.
+For example, Manubot intentionally does not rely on LaTeX, although users are free to export manuscripts to LaTeX themselves.
+In a future release, Pandoc [will support](https://github.com/greenelab/manubot-rootstock/pull/51#issuecomment-321949622) export to [Journal Article Tag Suite](https://jats.nlm.nih.gov/) (JATS).
+JATS is a standard XML format for scholarly articles that is used by publishers, archives, and text miners [@url:http://www.niso.org/standards/z39-96-2015/; @doi:10.6087/kcse.2014.1.99; @doi:10.1080/00987913.2012.10765464].
+For now however, the primary Manubot output is HTML intended to be viewed in a web browser.
+From the rendered HTML, Manubot also exports to PDF.
+
+Manubot performs continuous publication: every update to a manuscript's source is automatically reflected in the online outputs.
+The approach uses continuous integration (CI) [@doi:10.1038/550143a; @doi:10.1038/nbt.3780], specifically [Travis CI](https://travis-ci.org/) at the moment, to monitor for changes.
+When changes occur, the CI service rebuilds the manuscript.
+If the manuscript build process is error free, the CI service timestamps the manuscript and uploads the output files to the GitHub repository.
+Since the HTML manuscript is hosted using [GitHub Pages](https://pages.github.com/), the CI service automatically deploys the new manuscript version when it pushes the updated outputs to GitHub.
+
+For this article, the source GitHub repository is https://github.com/greenelab/meta-review.
+When this repository changes, Travis CI [rebuilds](https://travis-ci.org/greenelab/meta-review) the manuscript.
+If successful, the output is deployed back to GitHub (to dedicated [`output`](https://github.com/greenelab/meta-review/tree/output) and [`gh-pages`](https://github.com/greenelab/meta-review/tree/gh-pages) branches).
+As a result, https://greenelab.github.io/meta-review stays up to date with the latest HTML manuscript.
+
+Manubot uses [OpenTimestamps](https://opentimestamps.org/) to timestamps the HTML and PDF outputs on the Bitcoin blockchain before deploying to GitHub.
+This procedure allows one to retrospectively prove that a manuscript version existed prior to its blockchain-verifiable timestamp [@arxiv:1502.04015v1; @url:https://www.bgcarlisle.com/blog/2014/08/25/proof-of-prespecified-endpoints-in-medical-research-with-the-bitcoin-blockchain/; @url:http://blog.dhimmel.com/irreproducible-timestamps/; @url:http://git.dhimmel.com/bitcoin-whitepaper/].
+The implications for scientific writing are twofold.
+First, scientific precedence can now be indisputably established.
```

Comment:
Is that necessarily a good thing?  An [eLife article](https://doi.org/10.7554/eLife.28699) recently pushed back against scientific precedence.

Reply:
The implications are probably most important legally, when precedence can effect the outcome of an intellectual property dispute or some other question. Personally when evaluating science, I care for quality over novelty. For example, I may cite a more thorough and accessible study over an earlier study. In short, being able to establish precedence is potentially valuable for authors. We're not taking a position of the use of precedence or novelty when judging scientific output. The extent novelty should be used for evaluation (or in legal cases) is a separate question that we don't need to answer.